lib/operation.js

```diff
@@ -170,6 +170,44 @@ function blur (sigma) {
   return this;
 }
 
+/**
+ * Dilate the image.
```

Comment:
It might be nice to expand a little about what this means and mention things like "morphology" and the use of binary/two-tone/greyscale images being more suitable.